dnf/cli/cli.py

```
@@ -825,6 +825,7 @@ def configure(self, args):
 
         if opts.quiet:
             opts.debuglevel = 0
+            opts.errorlevel = 0
```

but then it will be hiding errors as well, no?